sdk/core/src/services/WalletStorageService.ts

```diff
@@ -304,18 +274,6 @@ export default class WalletStorageService {
     }
   }
 
-  async encryptAndSaveCredentials(data: unknown[], storageRegion?: string) {
-    const publicKeyBuffer = this._keysService.getOwnPublicKey()
-    const encryptedCredentials = []
-
-    for (const item of data) {
-      const cyphertext = await this._platformEncryptionTools.encryptByPublicKey(publicKeyBuffer, item)
-      encryptedCredentials.push(cyphertext)
-    }
-
-    return await this.saveCredentials(encryptedCredentials, storageRegion)
-  }
-
   async fetchAllDecryptedCredentials() {
```

Comment:
This method is used in `AffinityWallet.getCredentials` method in browser, expo, node, and react-native SDKs, and this is what API users actually use.

This means that all the API users, even after they upgrade to new SDK and fix all incompatibilities, will still only be reading from the old SDK, unless they replace calls to `AffinityWallet.getCredentials` with `AffinityWallet.getAllCredentials` (which does not even support handling `credentialShareRequestToken`.

One of the solutions would be to update all the specific `getCredentials` methods to call `WalletStorageService.getAllCredentials` instead of `fetchAllDecryptedCredentials`.

Then there is a problem with expo and react-native caching all fetched credentials in AffinityWallet object; I'll start a Slack discussion on that topic now.

Also, please make sure that all methods that only work with bloom vault are renamed to make this clear (to avoid confusion). For example, this method could be renamed to `fetchAllDecryptedCredentialsFromBloomVault`.

Reply:
This is still not addressed.

Please implement FTL-1755 as described in earlier comments:
* `AffinidiVaultStorageService` as a wrapper over `AffinidiVaultApiService` with all the required authorization / encryption / decryption / hashing logic (you already did this part);
* `BloomVaultStorageService` as a wrapper over `BloomVaultApiService` (it will basically be a copy of `WalletStorageService` prior to this PR)
* `WalletStorageService` with interface similar to `AffinidiVaultStorageService`, implementing the logic of "fetch single credential from AffinidiVault, if not found there, fetch it from BloomVault", "delete single credential from AffinidiVault, if not found there, delete it from BloomVault" and "fetch credentials by types from AffinidiVault, concatenate with filtered credentials from BloomVault".

And please make sure that everything builds and lint checks are passing successfully and all tests are passing successfully for dependent projects (`browser`, `expo`, `node`, `react-native`) as well.

Reply:
Yes, that is not finished yet, forgot move it to draft.